src/views/ThreeWProjectForm/schema.ts

```diff
@@ -0,0 +1,242 @@
+import {
+    PartialForm,
+    ArraySchema,
+    ObjectSchema,
+    nullValue,
+    addCondition,
+    undefinedValue,
+    requiredCondition,
+    requiredStringCondition,
+} from '@togglecorp/toggle-form';
+import { isDefined } from '@togglecorp/fujs';
+import {
+    positiveIntegerCondition,
+    positiveNumberCondition,
+} from '#utils/form';
+import {
+    OPERATION_TYPE_PROGRAMME,
+    OPERATION_TYPE_EMERGENCY,
+    PROGRAMME_TYPE_MULTILATERAL,
+    PROGRAMME_TYPE_DOMESTIC,
+    PROJECT_STATUS_COMPLETED,
+} from '#utils/constants';
+
+import type { paths } from '#generated/types';
+
+export type ProjectResponseBody = paths['/api/v2/project/{id}/']['put']['requestBody']['content']['application/json'];
+
+export type AnnualSplit = NonNullable<ProjectResponseBody['annual_split_detail']>[number] & {
+    client_id: string;
+};
+
+type ProjectFormFields = Omit<ProjectResponseBody, 'annual_split_detail'> & {
+    annual_split_detail: AnnualSplit[],
+};
```

Comment:
We can also use DeepReplace<ProjectResponseBody, OldAnnualSplit, NewAnnualSplit>

```
export type DeepReplace<T, A, B> = (
    T extends A
        ? B
        : (
            T extends (infer Z)[]
                ? DeepReplace<Z, A, B>[]
                : (

                    T extends object
                        ? { [K in keyof T]: DeepReplace<T[K], A, B> }
                        : T
                )
        )
)
```

Reply:
This is okay for now.